core/arch/proc.h

```diff
@@ -534,9 +549,10 @@ void
 proc_restore_fpstate(byte *buf);
 
 DR_API
-/** Returns whether AVX is enabled by both the processor and the operating system.
- * Even if the processor supports AVX, if the operating system does not enable
- * AVX state saving, then AVX instructions will fault.
+/**
+ * Returns whether AVX(2) is enabled by both the processor and the OS. Even
```

Comment:
I think "AVX (or AVX2)" would be clearer, leaving the subsequent references as just "AVX".

Reply:
Ok. Changed.